<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report a bug!</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font-weight: bold; color: #3399ff;</string>
       </property>
       <property name="text">
        <string>Subject:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ticket_title"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="cc_layout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="cc_label">
       <property name="styleSheet">
        <string notr="true">font-weight: bold; color: #3399ff;</string>
       </property>
       <property name="text">
        <string>CC:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="type_layout">
     <item>
      <widget class="QLabel" name="type_label">
       <property name="styleSheet">
        <string notr="true">font-weight: bold; color: #3399ff;</string>
       </property>
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="priority_layout">
     <item>
      <widget class="QLabel" name="priority_label">
       <property name="styleSheet">
        <string notr="true">font-weight: bold; color: #3399ff;</string>
       </property>
       <property name="text">
        <string>Priority</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font-weight: bold; color: #3399ff;</string>
     </property>
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bold">
       <property name="styleSheet">
        <string notr="true">font-weight: bold;
border: none;
background: none;</string>
       </property>
       <property name="text">
        <string>Bold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="italic">
       <property name="styleSheet">
        <string notr="true">font-style: italic;
border: none;
background: none;</string>
       </property>
       <property name="text">
        <string>Italic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="link">
       <property name="styleSheet">
        <string notr="true">border: none;
background: none;</string>
       </property>
       <property name="text">
        <string>Link</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="ticket_body">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">font-weight: bold; color: #3399ff;</string>
     </property>
     <property name="text">
      <string>Error Log:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="error_log">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="lower_layout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="screenshot">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 0;
border: none;
background: none;</string>
       </property>
       <property name="text">
        <string>Thumbnail</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="screen_grab">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1666</width>
         <height>1666</height>
        </size>
       </property>
       <property name="text">
        <string>Take a screenshot!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
